content/applications/sales/point_of_sale/restaurant/floors_tables.rst

```diff
@@ -2,91 +2,97 @@
 Floors and tables management
 ============================
 
-Add a floor
-===========
+Floor plan key
+==============
 
-Once your *Point of Sale* has been configured, select *Table
-Management* under :menuselection:`Point of Sale --> Configuration -->
-Point of Sale`. Then, click on *Floors* to create and name your floor and tables.
+Here are the keys to understanding what your floor plan view is telling you:
 
-.. image:: floors_tables/restaurant_03.png
-    :align: center
-    :alt: View of the table management feature. Way to manage and create floors for a pos
+.. image:: floors_tables/plan-understand.png
 
-.. image:: floors_tables/restaurant_04.png
-    :align: center
-    :alt: Backend view of a restaurant floor. Table name and number of sits for each table
+- Table 1: an order is placed and has been sent to the kitchen.
+- Table 3: an order of four items is placed and needs to be sent to the kitchen.
+- Tables 2, 4, and 5: the tables are available.
+- Tables 2, 4, and 5: the total capacity of these tables is respectively 2, 4, and 8 people.
+- Table 1: the table is taken by two people, out of a total capacity of 2.
+- Table 3: the table is taken by one person, out of a total capacity of 4.
 
-.. note::
-   Don’t forget to link your floor to your point of sale.
+Configuration
+=============
 
-Add tables
-==========
+From the POS settings
+---------------------
 
-To add tables, you can also open your PoS interface to see your
-floor(s).
+To create floors and tables from the backend, :ref:`access your POS' settings
+<configuration/settings>` and click :guilabel:`→ Floors` under the :guilabel:`Floors & Tables Map`
+category of the :guilabel:`Restaurant Mode` section.
 
-.. image:: floors_tables/restaurant_05.png
-    :align: center
-    :alt: View of the floors menu to manage several floors at the same time
+Then, click :guilabel:`New` to create a floor, give it a name, select the point of sales where that
+floor is to be displayed, and click :guilabel:`Add a line` to add some tables. Doing so opens a
+pop-up window to give your table a name, a number of seats, and to link it to an appointment
+resource. When satisfied with your table, click :guilabel:`Save & Close` or :guilabel:`Save & New`
+or click :guilabel:`Discard` to abort.
 
-Then, click on *Edit Mode* (pencil icon on the upper right corner) to be allowed to
-create, move, modify tables, etc.
+.. note::
+   - Your POS must be :ref:`configured as a bar or a restaurant <restaurant/configuration>` for that
+     section to be displayed.
+   - To create a map of your restaurant or bar reflecting your actual floor plan, you must open the
+     POS and :ref:`edit from the frontend <floors_tables/frontend>`.
```

Comment:
The POS needs to be opened and edited :ref:`from the frontend <floors_tables/frontend>`. to create a map of your restaurant or bar reflecting your actual floor plan.